body.tex

```diff
@@ -17,7 +17,7 @@ \section{Data}
 \subsection{Rubin DP1}
 \label{sec:data:dp1}
 
-The Rubin Observatory’s Data Preview 1 (DP1) dataset is the first public release of Rubin observatory imaging data processed through the LSST Science Pipelines, serving as a critical testbed for scientific and technical validation ahead of full LSST operations.  DP1 is based on observations from the LSST commissioning camera (LSSTComCam) and includes multi-band optical imaging in u, g, r, i, z and y filters over several square degrees of sky.  The dataset consists of processed images, source catalogs, and associated metadata, all formatted using the Rubin Data Butler system in the same way as full LSST data products.  Although smaller in scale than future LSST datasets, DP1 offers realistic photometric measurements, object detection, and data structures, making it an invaluable resource for developing and testing algorithms for tasks such as \photoz estimation, object classification, and data quality assessment.
+The Rubin Observatory’s Data Preview 1 (DP1) dataset is the first public release of Rubin observatory imaging data processed through the LSST Science Pipelines \cite{10.71929/rubin/2570325}, serving as a critical testbed for scientific and technical validation ahead of full LSST operations.  DP1 is based on observations from the LSST commissioning camera (LSSTComCam) and includes multi-band optical imaging in u, g, r, i, z and y filters over several square degrees of sky.  The dataset consists of processed images, source catalogs, and associated metadata, all formatted using the Rubin Data Butler system \cite{DMTN-229} in the same way as full LSST data products.  Although smaller in scale than future LSST datasets, DP1 offers realistic photometric measurements, object detection, and data structures, making it an invaluable resource for developing and testing algorithms for tasks such as \photoz estimation, object classification, and data quality assessment.
```

Comment:
`\citep`? 

It's much easier to do a github review of documentation if you use one sentence will line.

Reply:
`10.71929/rubin/2570325` is the doi for the Object catalog but it looks like you are attaching it to the pipelines comment. So maybe move that to the start as:

```latex
The Rubin Observatory's Data Preview 1 (DP1) Object catalog \citep{10.71929/rubin/2570325}...
```
Then use `\citep{PSTN-019}` for the pipelines ref.

PS for the Butler ref use https://ui.adsabs.harvard.edu/abs/2022SPIE12189E..11J/abstract not the tech note.